tests/iqm_client_test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020–2021 Cirq on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing IQM api client."""
+import pytest
+from requests import HTTPError
+from mockito import unstub
+import jsons
+from cirq_iqm.iqm_client import IQMBackendClient, IQMCircuit, QubitMapping
+from tests.coco_mock import mock_backend
+
+
+BASE_URL = "https://meetiqm.com/api/"
```

Comment:
Should use an example domain here too :)

Reply:
fixed all domains to https://example.com/